care/facility/api/viewsets/facility.py

```diff
@@ -183,3 +186,30 @@ class AllFacilityViewSet(
     filterset_class = FacilityFilter
     lookup_field = "external_id"
     search_fields = ["name", "district__name", "state__name"]
+
+
+class FacilityHubsViewSet(
+    mixins.RetrieveModelMixin,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    mixins.UpdateModelMixin,
+    mixins.DestroyModelMixin,
+    viewsets.GenericViewSet,
+):
```

Comment:
Isn't this same as ModelViewSet?

<img width="590" alt="image" src="https://github.com/coronasafe/care/assets/25143503/dc313345-07bb-4d88-9a1a-43d46d1537bd">


```suggestion
class FacilityHubsViewSet(ModelViewSet):
```